### PR TITLE
removes explicit ember-click-outside dependency.

### PR DESCRIPTION
**Pull request by stopfstedt**
we're getting that package via ilios-common.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -126,5 +125,6 @@
   "pre-commit": [
     "lint"
   ],
-  "private": true
+  "private": true,
+  "dependencies": {}
```
Comment (jrjohnson) [inline package.json:129]: On purpose?

Comment (stopfstedt) [reply package.json:129]: no, `npm uninstall ember-click-outside` bolted this one. remove?

Comment (stopfstedt) [reply package.json]: removed it.

Comment (jrjohnson) [reply package.json]: Yea, it's done that to me, maybe just leave it this time.

Comment (jrjohnson) [reply package.json]: or not 😁

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**